src/plans/create/steps/review/VirtualMachinesReviewSection.tsx

```diff
@@ -0,0 +1,84 @@
+import { type FC, useState } from 'react';
+import { useWatch } from 'react-hook-form';
+
+import ExpandableReviewSection from '@components/ExpandableReviewSection/ExpandableReviewSection';
+import {
+  Button,
+  ButtonVariant,
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  Modal,
+  ModalVariant,
+  useWizardContext,
+} from '@patternfly/react-core';
+import { useForkliftTranslation } from '@utils/i18n';
+
+import { planStepNames, PlanWizardStepId } from '../../constants';
+import { useCreatePlanFormContext } from '../../hooks';
+import { VmFormFieldId } from '../virtual-machines/constants';
+import VirtualMachinesTable from '../virtual-machines/VirtualMachinesTable';
+
+const VirtualMachinesReviewSection: FC = () => {
+  const { t } = useForkliftTranslation();
+  const { goToStepById } = useWizardContext();
+  const { control } = useCreatePlanFormContext();
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const vms = useWatch({
+    control,
+    name: VmFormFieldId.Vms,
+  });
+  const vmCount = Object.values(vms).length;
+
+  return (
+    <ExpandableReviewSection
+      title={planStepNames[PlanWizardStepId.VirtualMachines]}
+      onEditClick={() => {
+        goToStepById(PlanWizardStepId.VirtualMachines);
+      }}
+    >
+      <DescriptionList isHorizontal horizontalTermWidthModifier={{ default: '18ch' }}>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{t('Selected VMs')}</DescriptionListTerm>
+
+          <DescriptionListDescription>
+            <Button
+              isInline
+              variant={ButtonVariant.link}
+              onClick={() => {
+                setIsModalOpen(true);
+              }}
+            >
+              {t(`{{total}} virtual machine${vmCount > 1 ? 's' : ''} selected`, { total: vmCount })}
```

Comment:
please look on t with plurals, no need for the check for 's'

Reply:
I tried using i18n to handle this instead, but it looks like our parser does not handle this format appropriately:
```'{{total}} virtual machine{count, plural, one{} other{s}} selected'```

Looks like we'll need another dependency to handle this; `i18next-icu` and also update our parser config. If you'd like us to do this, I would prefer us handle it as a separate issue as it might require some debugging of our parser and some trial/error.